### PR TITLE
Add calculator component

### PR DESCRIPTION
**Pull request by bimbolabuari**
## Math Magician Calculator 👍 

 > This PR adds a Microverse project on developing a React class-based component for a calculator app, that demonstrates the knowledge of React components, ES6 syntax, import and export modules, JSX, CSS Grid Layout.

## Live Preview 🚀 

[Live Demo Link](https://deploy-preview-3--bimbola-mv-math-magician.netlify.app/)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,17 +1,11 @@
...
+import Calculator from './components/Calculator';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
   </React.StrictMode>,
   document.getElementById('root')
-);
-
```
Comment (vichuge) [inline src/index.js]: - [ ] Inside `index.js` you should call a component called `App.js` according to the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Once%20the%20Calculator%20is%20ready%2C%20import%20it%20and%20use%20it%20in%20your%20main%20component%20(App.js).). To use an `App.js` is a good practice inside a webpack project. 🐱 
- Remember, `App.js` should be a class-based component.
- `App.js` should have `Calculator.js` inside of it.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+function Calculator() {
```
Comment (vichuge) [inline src/components/Calculator.js]: - [ ] Please, Calculator should be a class-based component, right now is a function component. I kindly invite you to refactor it. In the next steps, you'll see why this is important for the project. 🌵

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] It looks like there are some things you could delete as unnecessary files, I invite you to do it. 🐶 
image
image

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 2 above)
Comment (bimbolabuari) [reply src/components/Calculator.js]: Thanks, Vichuge for the point out. I will make the changes right away.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @bimbolabuari  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**